.gitpod.yml

```
@@ -0,0 +1,27 @@
+tasks:
+  - before: nvm install 14
+    init: |
+      mkdir my-project
+      cp -R examples/magento-graphcms/. my-project
+      cd my-project
```

If we just want to spin up an example, it is also possible to just do:

```suggestion
      cd examples/magento-graphcms
```